docs/concepts/terminology.md

```diff
@@ -0,0 +1,215 @@
+composite resource as a database entry, while an XRD is a database schema. For
+those familiar with Kubernetes, the relationship is very similar to that between
+a Custom Resource Definition (CRD) and a Custom Resource (CR).
+
+A CompositeResourceDefinition is a distinct Crossplane API type, so Pascal case
+and fixed width fonts are appropriate.
+
+> There isn’t a direct analog to XRDs in the Helm ecosystem, but they’re a
+> little bit like the variable blocks in a Terraform module that define which
+> variables exist, whether those variables are strings or integers, whether
+> they’re required or optional, etc.
+
+### Managed Resource
+
+Managed resources are granular, high fidelity Crossplane representations of a
+resource in an external system - i.e. resources that are managed by Crossplane.
+Managed resources are what Crossplane enables platform teams to compose into
+higher level composite resources, forming an opinionated platform API. They're
+the building blocks of Crossplane.
+
+You’ll often hear three related terms used in the Crossplane ecosystem; composed
+resource, managed resource, and external resource. While there are some subtle
+contextual differences, these all broadly refer to the same thing. Take an
+RDSInstance for example; it is a managed resource. A distinct resource within
+the Crossplane API that represents an AWS RDS instance. When we make a
+distinction between the managed resource and an external resource we’re simply
+making the distinction between Crossplane’s representation of the thing (the
+`RDSInstance` in the Kubernetes API), and the actual thing in whatever external
+system Crossplane is orchestrating (the RDS instance in AWS's API). When we
+mention composed resources, we mean a managed resource of which a composite
+resource is composed.
+
+Managed resources are a class of resource, so avoid using Pascal case - “managed
+resource” not “ManagedResource”.
+
+> Managed resources are similar to Terraform resource blocks, or a distinct
+> Kubernetes resource within a Helm chart.
+
+### Package
+
+Packages extend Crossplane, either with support for new kinds of composite
+resources and claims, or support for new kinds of managed resources. There are
+two types of Crossplane package; configurations and providers.
+
+A package is not a distinct type in the Crossplane API, but rather a class of
+types. Therefore Pascal case is not appropriate.
+
+### Configuration
+
+A configuration extends Crossplane by installing conceptually related groups of
+XRDs and Compositions. Put otherwise, it configures the opinionated platform API
+that Crossplane exposes.
+
+A `Configuration` is a distinct type in the Crossplane API, therefore Pascal
+case and fixed width fonts are appropriate.
+
+### Provider
+
+A provider extends Crossplane by installing controllers for new kinds of managed
+resources. Providers typically group conceptually related managed resources; for
```

Comment:
maybe? feel free to keep as is if this also is out of scope
```suggestion
resources. Providers typically group conceptually related managed resources and their controllers; for
```

Reply:
Controllers are mentioned above here - I think that's sufficient.